src/main/drivers/pwm_output.h

```diff
@@ -112,6 +114,8 @@ typedef enum {
 #define PROSHOT_BASE_SYMBOL          24 // 1uS
 #define PROSHOT_BIT_WIDTH            3
 #define MOTOR_NIBBLE_LENGTH_PROSHOT  96 // 4uS
+
+#define DSHOT_TELEMETRY_DEADTIME     (2 * 30 + 10) // 2 * 30uS to switch lines plus 10us grace period
```

Comment:
To make this easier to maintain, and to be in line with best practices in Betaflight, this should be `DSHOT_TELEMETRY_DEADTIME_US` (same for `inputDirectionStampUs`, `dshotTelemetryDeadtimeUs`).